src/go/rpk/pkg/cli/cmd/wasm/deploy.go

```diff
@@ -113,25 +113,19 @@ func deploy(
 }
 
 func createHeaders(
-	action string, description string, content []byte,
+	action string, description string, content []byte, filename string,
 ) ([]sarama.RecordHeader, error) {
-	var headersResult []sarama.RecordHeader
-	// create simple struct for key string and value string
-	value := []struct {
-		key	string
-		value	string
-	}{
-		{key: "action", value: action},
-		{key: "description", value: description},
-	}
-	// create RecordHeader and append to headersResult
-	for _, v := range value {
-		keyH := []byte(v.key)
-		valueH := []byte(v.value)
-		headersResult = append(headersResult, struct {
-			Key	[]byte
-			Value	[]byte
-		}{Key: keyH, Value: valueH})
+	var headersResult []sarama.RecordHeader = []sarama.RecordHeader{
+		{
+			Key:	[]byte("action"),
+			Value:	[]byte(action),
+		}, {
+			Key:	[]byte("description"),
+			Value:	[]byte(description),
+		}, {
+			Key:	[]byte("id"),
```

Comment:
why don't we change this to filename, if that is what it really is?

Reply:
In your previous review, you wrote:
``Can we change the name of the key to id instead of filename? It is a bytes type which isn't human readable anyway.```

Reply:
Lets change it back! My fault, I replied on the original comment above